metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -1422,40 +1329,58 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
         yield from self.create_platform_mces(
             non_test_nodes,
             additional_custom_props_filtered,
-            manifest_nodes_raw,
+            all_nodes_map,
             DBT_PLATFORM,
             self.config.platform_instance,
         )
 
         yield from self.create_platform_mces(
             non_test_nodes,
             additional_custom_props_filtered,
-            manifest_nodes_raw,
+            all_nodes_map,
             self.config.target_platform,
             self.config.target_platform_instance,
         )
 
         yield from self.create_test_entity_mcps(
             test_nodes,
             additional_custom_props_filtered,
-            manifest_nodes_raw,
+            all_nodes_map,
         )
 
         if self.config.test_results_path:
             yield from DBTTest.load_test_results(
                 self.config,
                 self.load_file_as_json(self.config.test_results_path),
                 test_nodes,
-                manifest_nodes_raw,
+                all_nodes_map,
             )
 
         yield from self.stale_entity_removal_handler.gen_removed_entity_workunits()
 
+    def filter_nodes(self, all_nodes: List[DBTNode]) -> List[DBTNode]:
+        nodes = []
+        for node in all_nodes:
+            key = node.dbt_name
+
+            if not self.config.node_type_pattern.allowed(node.node_type):
```

Comment:
Do these need to be added to the report as filtered?

Reply:
the dbt report doesn't have a filtered object, so I'm just leaving that behavior as-is (this was just a refactor)

we'll add it to the backlog